base/math.jl

```diff
@@ -47,6 +47,8 @@ end
 Return `x` if `lo <= x <= hi`. If `x > hi`, return `hi`. If `x < lo`, return `lo`. Arguments
 are promoted to a common type.
 
+See also [`clamp!`](@ref), [`trunc`](@ref).
```

Comment:
```suggestion
See also [`clamp!`](@ref).
```

Then for the `clamp(x, T)::T` docstring, add:
```julia
See also [`trunc(T, x)`](@ref).
```

Reply:
Done in https://github.com/JuliaLang/julia/pull/38606/commits/273cd205f66934ad470b308259f705c79cf953d8, slightly enlarged...